purchases/src/main/kotlin/com/revenuecat/purchases/common/verification/SignatureVerificationMode.kt

```diff
@@ -6,21 +6,29 @@ internal sealed class SignatureVerificationMode {
     companion object {
         fun fromEntitlementVerificationMode(
             verificationMode: EntitlementVerificationMode,
-            signatureVerifier: SignatureVerifier? = null,
+            rootVerifier: SignatureVerifier? = null,
         ): SignatureVerificationMode {
             return when (verificationMode) {
                 EntitlementVerificationMode.DISABLED -> Disabled
                 EntitlementVerificationMode.INFORMATIONAL ->
-                    Informational(signatureVerifier ?: DefaultSignatureVerifier())
+                    Informational(IntermediateSignatureHelper(rootVerifier ?: DefaultSignatureVerifier()))
```

Comment:
we could update the "during feature beta" comment below since we're not calling this beta now

Reply:
Ahh good catch! Will update that